src/Aspire.Hosting/Publishing/ManifestPublishingContext.cs

```diff
@@ -105,12 +106,36 @@ public async Task WriteContainerAsync(ContainerResource container)
             }
         }
 
-        // Write volume details
+        // Write volume & bind mount details
         if (container.TryGetAnnotationsOfType<ContainerMountAnnotation>(out var mounts))
         {
-            var volumes = mounts.Where(mounts => mounts.Type == ContainerMountType.Named).ToList();
+            // Write out details for bind mounts
+            var bindMounts = mounts.Where(mounts => mounts.Type == ContainerMountType.Bind).ToList();
+            if (bindMounts.Count > 0)
+            {
+                // Bind mounts are written as an array of objects to be consistent with volumes
+                Writer.WriteStartArray("bindMounts");
+
+                foreach (var bindMount in bindMounts)
+                {
+                    Writer.WriteStartObject();
+
+                    Writer.WritePropertyName("source");
+                    Writer.WriteStringValue(PathNormalizer.NormalizePathForManifest(bindMount.Source));
```

Comment:
This should be a manifest relative path.

Reply:
yay path mapping

Reply:
This is definitely going to make testing harder as it will resolve differently on different machines for the paths that aren't under the AppHost project.

Reply:
I'll just remove the test cases for absolute paths for now.

Reply:
OK I updated this to use the existing method and removed the test cases that use fully qualified paths as those will obviously differ depending on where they're run.